src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -42,6 +54,22 @@ const AccessibilityRequestDetailPage = () => {
   const documents = data?.accessibilityRequest?.documents || [];
   const testDates = data?.accessibilityRequest?.testDates || [];
 
+  const removeRequest = () => {
+    mutate({
+      variables: {
+        input: {
+          id: accessibilityRequestId
+        }
+      }
+    }).then(() => {
+      history.push('/', {
```

Comment:
Can you only do the redirect if the response errors are empty? I think this is a pattern we might want to get better at. 
```
.then(response => {
  if (!response.error) {
    // page redirect stuff
  }
})
```